cmd/maya-apiserver/spc-watcher/select_disk_test.go

```diff
@@ -179,7 +180,23 @@ func TestNodeDiskAlloter(t *testing.T) {
 			0,
 			false,
 		},
+		// Test Case #8
+		"manualSPC8": {&v1alpha1.StoragePoolClaim{
+			Spec: v1alpha1.StoragePoolClaimSpec{
+				Type: "disk",
+				PoolSpec: v1alpha1.CStorPoolAttr{
+					PoolType: "mirrored",
+				},
+				Disks: v1alpha1.DiskAttr{
+					DiskList: []string{"disk1", "disk2", "disk3", "disk4"},
+				},
```

Comment:
Can we add one more test with: **5 disks and expectedDiskListLength: 4 with PoolType: mirrored** , which mean node having a 5 disk and having a pooltype mirrored  will select 4 disks out of 5 disks.

Reply:
@prateekpandey14 -- Done! Added a mirrored test case with 3 disks